lastuser_core/registry.py

```diff
@@ -126,17 +155,14 @@ def __delitem__(self, key):
 
 class LoginError(Exception):
     """External service login failure"""
-    pass
 
 
 class LoginInitError(Exception):
     """External service login failure (during init)"""
-    pass
 
 
 class LoginCallbackError(Exception):
     """External service login failure (during callback)"""
-    pass
```

Comment:
Are these classes just hanging now?

Reply:
`pass` is optional when there's a docstring. Autoflake removed it.